### PR TITLE
Fixed wrong copy

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description
Insert correct copy when timeline status is analog failure workflow
## List of changes proposed in this pull request

in notification utilities file delete control on generetadAarUrl

## How to test
login with PF, username: leonardo in dev enviroment and go on notification with iun : 
YDWX-JQGA-XVTL-202410-H-1

and visualize in timeline correct copy

### CODE REVIEW

File: packages/pn-commons/src/utility/notification.utility.ts [hunk 1]
```diff
@@ -309,6 +308,12 @@ export function getLegalFactLabel(
   legalFactType?: LegalFactType | 'AAR',
   legalFactKey?: string
 ): string {
+  console.log(
```
Comment (AndreaCimini90) [inline packages/pn-commons/src/utility/notification.utility.ts:311]: remove this

File: packages/pn-commons/src/utility/notification.utility.ts [hunk 2]
```diff
@@ -386,10 +391,7 @@ export function getLegalFactLabel(
       'detail.timeline.legalfact.analog-failure-delivery',
       'Deposito di avvenuta ricezione'
     );
-  } else if (
-    timelineStep.category === TimelineCategory.ANALOG_FAILURE_WORKFLOW &&
-    (timelineStep.details as AnalogWorkflowDetails).getGeneratedAarUrl
-  ) {
+  } else if (timelineStep.category === TimelineCategory.ANALOG_FAILURE_WORKFLOW) {
```
Comment (AndreaCimini90) [inline packages/pn-commons/src/utility/notification.utility.ts:394]: Why do you remove the (timelineStep.details as AnalogWorkflowDetails).getGeneratedAarUrl condition?

The TimelineCategory.ANALOG_FAILURE_WORKFLOW can occurs in several ways. Removing the check on generatedAarUrl you are telling that ALL the ANALOG_FAILURE_WORKFLOW must have the same legal fact label and this is wrong!
Only when we have generatedAarUrl, we have an AAR and we have to set this label, in the other cases the default "Attestazione opponibile a terzi" is good

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-commons/src/utility/notification.utility.ts (hunk 2 above)
Comment (leleOFA) [reply packages/pn-commons/src/utility/notification.utility.ts:394]: in our case the variable getAarUrl is never evaluated and therefore never enters the if

Comment (AndreaCimini90) [reply packages/pn-commons/src/utility/notification.utility.ts:394]: @leleOFA do you have checked the timeline element that corresponds to the ANALOG_FAILURE_WORKFLOW category?

Comment (leleOFA) [reply packages/pn-commons/src/utility/notification.utility.ts:394]: corresponds to that, in fact by removing the condition that I eliminated in the commit the right copy is inserted

Comment (leleOFA) [reply packages/pn-commons/src/utility/notification.utility.ts:394]: if I insert the condition without the generate but leaving the old one and adding a new one it inserts the copy correctly

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**